experimental/packages/exporter-trace-otlp-http/package.json

```diff
@@ -64,30 +64,30 @@
   },
   "devDependencies": {
     "@babel/core": "7.16.0",
+    "@jsdevtools/coverage-istanbul-loader": "3.0.5",
```

Comment:
Replace `istanbul-instrumenter-loader`  with `@jsdevtools/coverage-istanbul-loader` because IIL is unmaintained and has no plans to support webpack 5